act/workers/libs/crowdstrike_intel.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+from datetime import datetime
+from logging import debug, warning
+from typing import Iterator, List, Set
+
+from falconpy import Intel
+from pydantic import BaseModel, Field, validator
+from pydantic.typing import Literal  # type: ignore
+
+KILL_CHAIN_TYPE = Literal[
+    "reconnaissance",
+    "weaponization",
+    "delivery",
+    "exploitation",
+    "installation",
+    "c2",
+    "actiononobjectives",
+]
+
+
+class CrowdStrikeAPIError(Exception):
```

Comment:
Kan man her vurdere å lage støtte for å ta i mot erroren som kastes, slik at ting som feks status_code er tilgjengelig i Exceptionen (for logging eller printing feks). Litt som Hatching har gjort i Triage-klienten sin her:

https://github.com/hatching/triage/blob/62eeb67cce329d12008ad9f6a4f60ba660da1e0e/python/triage/client.py#L553

Reply:
Thanks - I have implemented logic similar to this in the current PR now!